src/plugins/core/plugin_minify_html_literals.ts

```diff
@@ -10,43 +10,43 @@ interface IPluginMinifyHtmlLiterals {}
  * @param options
  */
 export function pluginMinifyHtmlLiterals(...options: IPluginMinifyHtmlLiterals[]) {
-  // const useroptions = options;
+  const useroptions = options;
 
   return (ctx: Context) => {
-    // ctx.ict.on('bundle_resolve_module', props => {
-    //   const { module } = props;
-    //   if (module.isExecutable()) {
-    //     // verify if module is installed
-    //     if (!ctx.isInstalled('minify-html-literals')) {
-    //       ctx.fatal(`Fatal error when capturing ${module.props.absPath}`, [
-    //         'Module "minify-html-literals" is required, Please install it using the following command',
-    //         'npm install minify-html-literals --save-dev',
-    //       ]);
-    //       return;
-    //     }
-    //     // check if we have any html`` or css`` template tags
-    //     if (/\html`|css`/.test(module.contents)) {
-    //       ctx.log.info('pluginMinifyHtmlLiterals', 'minifying $file', {
-    //         file: module.props.absPath,
-    //       });
-    //       // https://github.com/asyncLiz/minify-html-literals
-    //       let newcontent;
-    //       try {
-    //         const { minifyHTMLLiterals } = require('minify-html-literals');
-    //         newcontent = minifyHTMLLiterals(module.contents, ...useroptions);
-    //       } catch (e) {
-    //         console.log(e);
-    //       }
-    //       if (newcontent && newcontent.code) {
-    //         module.contents = newcontent.code;
-    //       } else {
-    //         ctx.log.error('pluginMinifyHtmlLiterals - failed $file', {
-    //           file: module.props.absPath,
-    //         });
-    //       }
-    //     }
-    //   }
-    //   return props;
-    // });
+    ctx.ict.on('bundle_resolve_module', props => {
+      const { module } = props;
+
+      // verify if module is installed
+      if (!isNodeModuleInstalled('minify-html-literals')) {
+        ctx.fatal(`Fatal error when capturing ${module.absPath}`, [
+          'Module "minify-html-literals" is required, Please install it using the following command',
+          'npm install minify-html-literals --save-dev',
+        ]);
+        return;
+      }
+      // check if we have any html`` or css`` template tags
+      if (/\html`|css`/.test(module.contents)) {
```

Comment:
gotta check for `module.isExecutable` here

Reply:
K, ill have a look tomorrow after work

Reply:
fixed... wil be very late if I wait until tomorrow :-)
time for bed :-)